src/MenampilkanList.jsx

```diff
@@ -2,15 +2,21 @@ import { useState } from 'react';
 import { Navigation } from 'swiper/modules';
 import { Swiper, SwiperSlide } from 'swiper/react';
 
-// Import style swiper elements
+import React from "react";
+import { ShimmerPostItem } from "react-shimmer-effects";
+
 import 'swiper/css';
 import 'swiper/css/pagination';
 import 'swiper/css/navigation';
 
 export default function AkanDihapus() {
     const [data, setData] = useState([]);
+    const [status, setStatus] = useState(false)
+
+
     function getData() {
         const inputan = document.getElementById('inputan');
+        setStatus(true)
         fetch(`http://www.omdbapi.com/?s=${inputan.value}&apikey=aebedaef`)
             .then(response => {
```

Comment:
buatkan kelas constant untuk variable" yang di sering atau berulang kali dipakai.

```suggestion
        fetch(`${BASE_URL}/?s=${inputan.value}&apikey=${API_KEY}`)
            .then(response => {
```